packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/__tests__/useLocationDetailView.spec.tsx

```diff
@@ -367,61 +378,74 @@ describe('useLocationDetailView', () => {
   });
 
   it('should set all file items as selected', () => {
-    const mockDataState = {
-      data: { items: testData, nextToken: undefined },
-      message: '',
-      hasError: false,
-      isLoading: false,
-    };
-
     useStoreSpy.mockReturnValue([
       {
         ...testStoreState,
-        locationItems: {
-          fileDataItems: [fileItem, fileItem, fileItem, fileItem],
-        },
+        locationItems: { fileDataItems: undefined },
       },
       mockDispatchStoreAction,
     ]);
-    useActionSpy.mockReturnValue([mockDataState, jest.fn()]);
+
+    const mockDataState = {
+      data: {
+        items: [folderDataOne, fileDataOne, fileDataTwo],
+        nextToken: undefined,
+      },
+      message: '',
+      hasError: false,
+      isLoading: false,
+    };
+
+    useDataStateSpy.mockReturnValue([mockDataState, jest.fn()]);
 
     const { result } = renderHook(() => useLocationDetailView());
-    const state = result.current;
+    const { onSelectAll } = result.current;
 
-    state.onSelectAll();
+    onSelectAll();
 
     expect(mockDispatchStoreAction).toHaveBeenCalledWith({
-      type: 'RESET_LOCATION_ITEMS',
+      type: 'SET_LOCATION_ITEMS',
+      items: [fileDataOne, fileDataTwo],
```

Comment:
Was asserting the opposite behavior of the test description, same issue with the test case immediately below